### DISCUSSION TITLE
Hvordan håndtere lang knappetekst?

### DISCUSSION THREAD

**Discussion by lfbergee** [fremtind/jokul#2287] [Spørsmål]
**Feilbeskrivelse**
Lange tekster flyter utenfor knappen

**For å reprodusere**
https://codesandbox.io/s/hidden-leaf-hoj71?file=/src/App.js

**Forventet oppførsel**
Teksten flyter ikke utenfor knappen

**Skjermbilder**
![Screenshot from 2020-08-19 09-49-22](https://user-images.githubusercontent.com/46530368/90619243-48af7080-e211-11ea-8041-d948267507c0.png)

**Comment by Steinop** [comment]
Dette bør rettes. Samtidig er det viktig å påpeke at tekst i knapper bør være så korte og konsise som mulig. 
Innspill: Ville det vært tydeligere for brukerne å hatt en +knapp når man f. eks skal legge til flere sykdommer?

**Comment by Mikkelinski** [comment]
Enig med Stein Olav.  Jeg har fortsatt tilgode å se en knapp som ikke kan kortes ned tilstrekkelig (overnevnte eksempel inkludert) - men jeg er enig at vi trenger en håndtering dersom uhellet er ute.

@Steinop skal vi ta plus knapp spørsmålet til teams hvor flere designere får det med seg og kan veie inn?

**Comment by Steinop** [comment]
Lag gjerne en poll på dette @Mikkelinski. Flyt har allerede en pluss-knapp som kan brukes som utgangspunkt. Tertiærknappen med + foran teksten kan også være et utgangspunkt avhengig av hvor mye fokus knappen trenger.

**Comment by wkillerud** [comment]
Ble det en konklusjon på dette med tekstbrytning i knapper?

Lengden på tekstene begrenses i utgangspunktet av bredden på containeren. Klart, på de smaleste mobilskjermene kan det være en utfordring slik at vi _kanskje_ burde ha støtte for tekstbrytning for sikkerhets skyld?

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/1223410/132685765-5a5ffd93-1e65-4ca6-9823-95ed5b8cec04.png">
